node-net-manager/handlers/deployment.go

```diff
@@ -17,6 +17,7 @@ type ContainerDeployTask struct {
 	ServiceName    string `json:"serviceName"`
 	Instancenumber int    `json:"instanceNumber"`
 	PortMappings   string `json:"portMappings"`
+	IsUnikernel    bool   `json:"isUnikernel,omitempty"`
```

Comment:
Instead of a bool, what if we use an enum with types:
-container
-unikernel

Then, in the future, implementing networking for a new virt will require us to just add value. What do you think?